### ISSUE TITLE
Pivot tables result in errors

### ISSUE THREAD

**Issue by nhusby** [b3nl/laravel-mwb-model#8]
When pivot tables are specified in either command line or MySQL Workbench files, it causes errors preventing operation.  

I attempted to fix it, but I inevitably gave up in favor of handling pivot tables manually.  It looks like the issue stems from inconsistent use of model names, table names, and ids.

**Comment by b3nl** [comment]
yep, some pivots are not working correctly  (#1). I am still not satisfied with my work, especially the pivot part.
